pandas/io/formats/latex.py

```diff
@@ -140,8 +139,8 @@ def pad_empties(x):
                     buf.write("\\endhead\n")
                     buf.write("\\midrule\n")
                     buf.write(
-                        "\\multicolumn{{{n}}}{{r}}{{{{Continued on next "
-                        "page}}}} \\\\\n".format(n=len(row))
+                        f"\\multicolumn{{{len(row)}}}{{r}}"
+                        "{{Continued on next page}} \\\\\n"
```

Comment:
why do two sets of braces need to be removed?

Reply:
This actually got me as well. 2 sets of braces are needed for the final string, since I remove the `.format` and this becomes a regular string there is no need for the "escape" sequence of 2 braces to show 1. Is that or 4 braces with f string